backend/src/main/java/ai/giskard/service/PushService.java

```diff
@@ -0,0 +1,36 @@
+package ai.giskard.service;
+
+import ai.giskard.ml.MLWorkerClient;
+import ai.giskard.service.ml.MLWorkerService;
+import ai.giskard.worker.*;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@Service
+//@Transactional
```

Comment:
It's rightfully not transactional since it's communicating with the ML Worker inside, you can remove the comment